inc/class-manage-multisite.php

```diff
@@ -0,0 +1,184 @@
+		add_action( 'admin_enqueue_scripts', array( __CLASS__, 'select_site_load_scripts' ) );
+		add_action( 'wp_ajax_select_site_get_results', array( __CLASS__, 'select_site_ajax' ) );
+		add_action( 'wp_before_admin_bar_render', array( __CLASS__, 'remove_pmpro_adminbar' ), 999 );
+	}
+
+	/**
+	 * [select_site_load_scripts description]
+	 *
+	 * @return [type] [description]
+	 */
+	public static function remove_pmpro_adminbar() {
+		global $wp_admin_bar;
+		$id = 'paid-memberships-pro';
+		$wp_admin_bar->remove_menu( $id );
+	}
+
+	/**
+	 * [select_site_load_scripts description]
+	 *
+	 * @return [type] [description]
+	 */
+	public static function select_site_load_scripts() {
+		$screen = get_current_screen();
+		if ( 'toplevel_page_pmpro-membership' === $screen->id || 'memberships_page_pmpro-multisite-membership' === $screen->id ) {
+			wp_enqueue_style( 'multisite-membership', plugin_dir_url( __FILE__ ) . 'css/multisite-membership.css' );
+		}
+		wp_enqueue_script( 'select-site', plugin_dir_url( __FILE__ ) . 'js/select-site.js', array( 'jquery' ) );
+		wp_localize_script(
+			'select-site', 'select_site_vars', array(
+				'select_site_nonce' => wp_create_nonce( 'select-site-nonce' ),
+			)
+		);
+	}
+
+	/**
+	 * [pmpro_multisite_membership_page description]
+	 *
+	 * @return [type] [description]
+	 */
+	public static function pmpro_multisite_membership_page() {
+		global $wpdb;
+		self::pmpro_membership_header();
+		?>
+		<div class="wrap">
+		<h2><?php esc_attr_e( 'PMPro Multisite Membership', 'selectsite' ); ?></h2>
+		<p>You have activated the <strong>Multisite Membership Add On</strong> on this site, which means that you will be using PMPro settings from another site in your Network to control site access.</p>
+
+		<p>In order to finish setting up the Multisite Membership Add On, you'll need to check that you have the proper prefix for the site controlling the settings in wp-config.php. Select the site which you will use as the Main site and click the button to get the prefix.</p>
+		<form id="select-site-form" action="" method="POST">
+			<div><strong><label>Select PMPro Domain</strong>
+				<?php echo self::render_sites_dropdown(); ?></label>
+				<input type="submit" name="select-site-submit" id="select_site_submit" class="button-primary" value="<?php esc_attr_e( 'Get Site Prefix', 'selectsite' ); ?>"/>
+				<img src="<?php echo esc_url( admin_url( '/images/wpspin_light.gif' ) ); ?>" class="waiting" id="select_site_loading" style="display:none;"/>
+			</div>
+		</form>
+		<div id="select_site_results"></div>
+
+		<p>You'll know that you have your prefix defined correctly when the value above and the value below match. Currently your prefix is:</p>
+		<h4><?php echo '<pre>define( \'PMPRO_NETWORK_MAIN_DB_PREFIX\', \'' . PMPRO_NETWORK_MAIN_DB_PREFIX . '\' );</pre>'; ?></h4>
+		</div>
+		<?php
+	}
+
+	/**
+	 * [select_site_ajax description]
+	 *
+	 * @return [type] [description]
+	 */
+	public static function select_site_ajax() {
```

Comment:
If the AJX is working, great. We may not need the updated settings page to be AJAX. If it just works as a simple submitted form, let's do that to keep it simple.